src/store/selectors/auth.ts

```diff
@@ -0,0 +1,17 @@
+import { RootState } from 'app/store';
+
+export function isLoading(state: RootState) {
+  return state.auth.isLoading;
+}
+
+export function isLoggedIn(state: RootState) {
+  return state.auth.isLoggedIn;
+}
+
+export function getUser(state: RootState) {
+  return state.auth.user;
+}
+
+export function getError(state: RootState) {
+  return state.auth.error;
```

Comment:
it's a matter of personal preference but what do you think about arrow function notation? it's shorter but what do you think about readability?
```
export const getError = (state: RootState) => state.auth.error
```